MeasureAuthoringTool/mat-drupal
# Initialize drupal-composer project.

## Description
Created base drupal project via composer. Added docker-compose config to ease local development. Modified Dockerfile to include `drush`.

## To Test
README.md was updated with some instructions, but this should get the site up and running:

1. Install composer dependencies `composer install`
2. Copy `.env.example` to `.env` and provide some values for the MySQL variables. Doesn't really matter
what you use, except that MYSQL_HOSTNAME should match docker-compose service name for MySQL ("database").
2. Start docker-compose `docker-compose up -d --build`
3. Run drupal installer `docker-compose exec drupal drush si`. You'll have to provide some values here for MySQL username and password, etc.
